### PR TITLE
Updates Pages functions docs

### PR DESCRIPTION
**Pull request by jkup**

### CODE REVIEW

File: products/pages/src/content/platform/functions.md [hunk 1]
```diff
@@ -441,18 +445,17 @@ $ npx wrangler pages dev --help
...
-# KV namespace to bind
-$ npx wrangler pages dev dist  --kv NAMESPACE
-
 # Bind variable/secret (KEY=VALUE)
-$ npx wrangler pages dev dist --binding ENV_NAME=\"ENV_VALUE"
-
-# Durable Object to bind (NAME=CLASS)
-$ npx wrangler pages dev dist  --do NAME=\"CLASS"
-
+$ npx wrangler pages dev dist --binding ENV_NAME=\"ENV_VALUE\"
```
Comment (Ekwuno) [inline products/pages/src/content/platform/functions.md:458]: We already merged a PR for this bit. https://github.com/cloudflare/cloudflare-docs/pull/3181

**Comment by Ekwuno** [comment]
LGTM!

File: products/pages/src/content/platform/functions.md [hunk 2]
```diff
@@ -360,18 +359,24 @@ To add environment variables, go to **Account Home** > **Pages** > **your Pages
...
-For example, you can connect [Sentry](https://www.sentry.io/) to your application middleware using [Toucan js](https://github.com/robertcepa/toucan-js) and access your DSN in your component. 
+For example, you can run `npx wrangler pages dev dist --binding COLOR=\"BLUE"` and then:
 
+```js
+export async function onRequest({ env }) {
+  return new Response(env.COLOR);
+}
+```
+
+Here is a real world example of using environment variables inside a middleware function. To connect [Sentry](https://www.sentry.io/) to a Cloudflare Worker, you can use [Toucan js](https://github.com/robertcepa/toucan-js) and access your Sentry Data Source Name (DSN) in your function.
```
Comment (deadlypants1973) [inline products/pages/src/content/platform/functions.md]:
```suggestion
Here is a real-world example of using environment variables inside a middleware function. To connect [Sentry](https://www.sentry.io/) to a Cloudflare Worker, you can use [Toucan js](https://github.com/robertcepa/toucan-js) and access your Sentry Data Source Name (DSN) in your function.
```

**Comment by jkup** [comment]
@Ekwuno @deadlypants1973 can I get a 🟢 review from one of you so I can merge?

File: products/pages/src/content/platform/functions.md (hunk 1 above)
Comment (jkup) [reply products/pages/src/content/platform/functions.md:458]: I was just adding a \= at the very end I think there is a typo in prod right now. I think the diff just made it look super weird cause I moved them into a different order!

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**